install.sh

```diff
@@ -162,12 +162,20 @@ else
 fi
 
 # Choose the right GHC download
+case `uname -m` in
+  i386)   GHC_CPU=i386;;
+  i686)   GHC_CPU=i386;;
+  x86_64) GHC_CPU=x86_64;;
+  amd7)   GHC_CPU=amd7;;
+esac
+
+## then
 if /sbin/ldconfig -p | grep -q libgmp.so.10; then
-  GHC_ARCH=`uname -m`-deb8-linux
+  GHC_ARCH=$GHC_CPU-deb8-linux
```

Comment:
It's not really necessary here, but I recommend quoting shell variable interpolation as a habit to avoid all sorts of nasty shell gotchas:

```bash
GHC_ARCH="${GHC_CPU}-deb8-linux"
```